src/dune/dune.py

```diff
@@ -105,7 +105,7 @@ def create_node(self, nod):
 
     def get_current_nodeos_version(self):
         stdout, stderr, exit_code = self._docker.execute_cmd(['nodeos', '--version'])
-        return stdout[1:].split('.-')
+        return re.split(r"[.-]", stdout[1:-1])
```

Comment:
```suggestion
        return re.split(r"[.-+]", stdout[1:-1])
```

After commenting about semver prerelease (`-`), I later realized  build (`+`) would be appropriate to split on as well. Think of this as a nice to have, not a must have.

Reply:
`.-+` gives error `re.error: bad character range .-+ at position 1`
but `+.-` seem to work which is interesting.
Anyway I have added it